From fc30088a1543cf2ce41725433cb462851e2e70f9 Mon Sep 17 00:00:00 2001
From: Garrett Brown <eigendebugger@gmail.com>
Date: Mon, 11 Apr 2022 12:11:48 -0700
Subject: [PATCH 1/2] Fix build error

Error was:

 MPU6050.cpp:3384:9: error: 'uint' was not declared in this scope
       for(uint i = 0; i < 6; i++){
           ^~~~
---
 Arduino/MPU6050/MPU6050.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Arduino/MPU6050/MPU6050.cpp b/Arduino/MPU6050/MPU6050.cpp
index edf478e..35bcebc 100644
--- a/Arduino/MPU6050/MPU6050.cpp
+++ b/Arduino/MPU6050/MPU6050.cpp
@@ -3381,7 +3381,7 @@ int16_t * MPU6050_Base::GetActiveOffsets() {
         I2Cdev::readWords(devAddr, AOffsetRegister+6, 1, (uint16_t *)(Data+2), I2Cdev::readTimeout, wireObj);
     }
     I2Cdev::readWords(devAddr, 0x13, 3, (uint16_t *)(Data+3), I2Cdev::readTimeout, wireObj);
-    for(uint i = 0; i < 6; i++){
+    for(unsigned int i = 0; i < 6; i++){
         offsets[i] = Data[i];
     }
     return offsets;
-- 
2.17.1

